README.md

```diff
@@ -13,7 +13,10 @@ Napa.js requires C++ compiler that supports [C++14](https://en.wikipedia.org/wik
     * Windows: Visual C++ Build Tools [2015](http://landinghub.visualstudio.com/visual-cpp-build-tools) / [2017](https://www.visualstudio.com/downloads/#build-tools-for-visual-studio-2017), or [Visual Studio](https://www.visualstudio.com/vs/cplusplus/)
     * Linux: [via Apt-get](https://askubuntu.com/questions/618474/how-to-install-the-latest-gcurrently-5-1-in-ubuntucurrently-14-04)
     * OSX: `xcode-select --install`
-* Install CMake: [Download or build](https://cmake.org/install/), [Apt-get (Linux/Ubuntu)](https://askubuntu.com/questions/355565/how-to-install-latest-cmake-version-in-linux-ubuntu-from-command-line)
+* Install CMake: 
+    * Linux: [via Apt-get](https://askubuntu.com/questions/355565/how-to-install-latest-cmake-version-in-linux-ubuntu-from-command-line)
+    * OSX: `brew install cmake`
```

Comment:
Thanks for contributing! 

`brew` is not installed on OSX by default, maybe it's better to point to an external link that contains complete steps. I found one here:
http://macappstore.org/cmake/